code/src/utest/plugin/lsttokens/datacontrol/RequiredTokenTest.java

```diff
@@ -112,7 +115,7 @@ public void testValidStringNo()
 		assertNull(cd.getRequired());
 		assertTrue(token.parseToken(context, cd, "NO").passed());
 		assertNotNull(cd.getRequired());
-		assertFalse(cd.getRequired().booleanValue());
+		Assertions.assertFalse(cd.getRequired().booleanValue());
```

Comment:
This feels inconsistent - assertNotNull is statically imported, but this is directly used?

Reply:
likely just missed it in the migration.